server/src/main/java/org/candlepin/dto/StandardTranslator.java

```diff
@@ -232,8 +230,8 @@ public StandardTranslator(ConsumerTypeCurator consumerTypeCurator,
             new org.candlepin.dto.manifest.v1.BrandingTranslator(),
             Branding.class, org.candlepin.dto.manifest.v1.BrandingDTO.class);
         this.registerTranslator(
-            new org.candlepin.dto.manifest.v1.ProductBrandingTranslator(),
-            ProductBranding.class, org.candlepin.dto.manifest.v1.BrandingDTO.class);
+            new org.candlepin.dto.manifest.v1.BrandingTranslator(),
```

Comment:
Same here, the previous line already registers the translator.

Reply:
Correcting it.